### PR TITLE
Theme bm linear progress bar

### PR DESCRIPTION
**Pull request by adicicurel**
### 🔦 Summary

Adding LinearProgressBar to Business Dashboard theme 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6136

File: src/Themes/businessDashboard/components/LinearProgressBar.st.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+:import {
+  -st-from: "../foundations/colors.st.css";
+  -st-named: B05;
+}
+
+.root {
+  -st-extends: LinearProgressBar;
+}
+
+.root:skin(standard)::barForeground {
```
Comment (shlomitc) [inline src/Themes/businessDashboard/components/LinearProgressBar.st.css:15]: need to add also `background: D60` for the barBackground

File: src/Themes/businessDashboard/docs/components/LinearProgressBar.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const playground = `
+<ThemeProvider theme={theme({ active: true })}>
```
Comment (shlomitc) [inline src/Themes/businessDashboard/docs/components/LinearProgressBar.js:2]: can't we simply set `active: true` as the default value? That would make code examples much cleaner

File: src/LinearProgressBar/test/LinearProgressBar.visual.js [hunk 3]
```diff
@@ -185,3 +187,46 @@ interactiveTests.forEach(({ describe, its }) => {
...
+
+export const runTests = (
+  { themeName, testWithTheme } = { testWithTheme: i => i },
+) => {
+  visualize(`${themeName ? `${themeName}|` : ''}LinearProgressBar`, () => {
+    story('Skins', () => {
+      Object.values(SKINS).forEach(skin => {
+        snap(skin, () => testWithTheme(renderSkin(skin)));
+      });
+    });
```
Comment (shlomitc) [inline src/LinearProgressBar/test/LinearProgressBar.visual.js:228]: Why did you need to add another test phase? There are already tests for all the different themes: https://github.com/wix/wix-style-react/pull/6136/files#diff-410caa42a341168a81e4bafe9b20528c389e0c66a09602df5a42e1d20ee6789bR72

Also, by exposing `runTests` only for this section, it means other tests will not run for your theme which isn't consistent with the other theme tests that we created.

File: src/LinearProgressBar/test/LinearProgressBar.visual.js [hunk 4]
```diff
@@ -6,6 +6,8 @@ import { linearProgressBarTestkitFactory } from '../../../testkit';
 import { SKINS } from '../constants';
 import { Layout, Cell } from '../../Layout';
 import Box from '../../Box';
+import Heading from '../../Text';
```
Comment (shlomitc) [inline src/LinearProgressBar/test/LinearProgressBar.visual.js:9]: not needed

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: src/Themes/businessDashboard/docs/components/LinearProgressBar.js (hunk 2 above)
Comment (dimafeldman) [reply src/Themes/businessDashboard/docs/components/LinearProgressBar.js]: yep, good idea. adjusted

Comment (adicicurel) [reply src/Themes/businessDashboard/docs/components/LinearProgressBar.js]: it already have a default active: true, just remove it from the docs components

**Review by shlomitc — APPROVED** [review]

File: src/Themes/businessDashboard/components/LinearProgressBar.st.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  -st-named: G20, D60;
+}
+
+.root {
+  -st-extends: LinearProgressBar;
+  padding: 0 0 0 8px;
+  height: 16px;
+}
+
+.root:skin(standard)::barForeground {
```
Comment (shlomitc) [inline src/Themes/businessDashboard/components/LinearProgressBar.st.css:17]: @dimafeldman - now seeing the visual tests failing - the overrides should be applied only for the `success` state and not for the standard one. can you please fix it?

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: src/Themes/businessDashboard/docs/components/LinearProgressBar.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export const playground = `
+<ThemeProvider theme={theme()}>
+  <LinearProgressBar value={30}/>
```
Comment (shlomitc) [inline src/Themes/businessDashboard/docs/components/LinearProgressBar.js:3]:
```suggestion
  <LinearProgressBar skin="success" value={30}/>
```

**Pull request merged**